project.clj

```diff
@@ -18,14 +18,15 @@
                  [puppetlabs/kitchensink ~ks-version]
                  [puppetlabs/certificate-authority "0.6.0"]
                  [puppetlabs/http-client "0.4.0"]
-                 [org.jruby/jruby-core "1.7.15" :exclusions [com.github.jnr/jffi com.github.jnr/jnr-x86asm]]
+                 [org.jruby/jruby-core "1.7.19"
+                  :exclusions [com.github.jnr/jffi com.github.jnr/jnr-x86asm]]
+                 [com.github.jnr/jffi "1.2.8"]
```

Comment:
https://github.com/jruby/jruby/blob/1.7.19/pom.xml#L75 shows a jffi.version of "1.2.7".  With the 1.7.19 tag selected in a local checkout of jruby src, "mvn dependency:tree -Dverbose" includes the following in the output:

~~~~
...
[INFO] org.jruby:jruby-core:jar:1.7.19
...
[INFO] +- com.github.jnr:jffi:jar:1.2.7:compile
[INFO] +- com.github.jnr:jffi:jar:native:1.2.7:compile
~~~~

A "lein deps \:tree" and "lein test" both worked okay for me against your branch with a project.clj modified to use:

~~~~
[com.github.jnr/jffi "1.2.7"]
[com.github.jnr/jffi "1.2.7" :classifier "native"]
~~~~

I don't have any evidence that 1.2.8 is harmful to use with jruby 1.7.19.  I was just thinking if it looks like that's what jruby 1.7.19 is using by default that it might be better to just stick with that.